auth_token/utils.py

```diff
@@ -360,7 +360,9 @@ def get_user_from_token(token):
     if token:
         backend_path = token.backend
         if backend_path in django_settings.AUTHENTICATION_BACKENDS and token.is_authenticated:
-            active_takeover_id = token.active_takeover.user_id if token.active_takeover else None
+            active_takeover_id = token.active_takeover.user_id if (
+                settings.TAKEOVER_ENABLED and token.active_takeover
+            ) else None
```

Comment:
There is called property active_takeover twice (not your fault)

in the logout is similar fail

![image](https://user-images.githubusercontent.com/424409/189289410-ce971f9f-8f83-4b44-8f1a-f038d70ba8bb.png)

this is more dangerous, because someone can create new takeover and different object will be logged off than was get in the function. Please change property to cached_property in active_takeover method.

sorting is also missing in the queryset :(
```

@cached_property
def active_takeover(self):
     return self.user_takeovers.filter(is_active=True).last('changed_at', 'id')


```